### PR TITLE
Add support for Github (GitHub Actions)

### PR DESCRIPTION
**Pull request by xakraz**
🙏 Feedback I am a beginner in Glue

## WHAT

Add Github in CI

## WHY

We are using Github actions for Terraform

## REF

https://help.github.com/en/articles/virtual-environments-for-github-actions
https://developer.github.com/v3/repos/commits/#list-pull-requests-associated-with-commit
https://github.com/google/go-github

### CODE REVIEW

**Comment by b4b4r07** [comment]
@xakraz 

Please fix conflicts and do `go fmt` first!

**Comment by xakraz** [comment]
@b4b4r07 I will do and work on it better by the end of the week.
🙏 for your time and work.

File: ci.go [hunk 1]
```diff
@@ -101,3 +102,28 @@ func jenkins() (ci CI, err error) {
...
 	return ci, err
 }
+
+func github() (ci CI, err error) {
+	ci.PR.Number = 0
+	ci.PR.Revision = os.Getenv("GITHUB_SHA")
+	ci.URL = ""
+	
+	# Get info from GitHub API
+	github_repository_split = strings.Split(os.Getenv("GITHUB_REPOSITORY"), "/")
```
Comment (b4b4r07) [inline ci.go]: Please use camelCase in this Go code. Other parts are also the same.

File: ci.go [hunk 2]
```diff
@@ -101,3 +102,28 @@ func jenkins() (ci CI, err error) {
...
+	pr, resp, err := github_client.RepositoryCommit.GetCommitSHA1(context.Background(), github_owner, github_repository_name, "GITHUB_SHA")
+        if err != nil {
+	  return ci, fmt.Errorf("Error when querying GitHub for commit %v", ci.PR.Revision)
+	}
+	if pr == "" {
+		return ci, nil
+	}
+	
+	ci.URL = pr[0]["html_ur"]
+        ci.PR.Number = pr[0]["number"]
```
Comment (b4b4r07) [inline ci.go]: Indent is still wrong. Please do `go fmt` again!

**Comment by b4b4r07** [comment]
Got stale. Once close.

**Pull request closed without merging**